assets/docs/configuration/transformations/snowplow-builtin/spGtmssPreview-full-example.hcl

```diff
@@ -0,0 +1,6 @@
+transform {
```

Comment:
We now need a minimal example for this tranformation, since there are options. (The minimal example is just the old example with a filename that matches the naming convention)